db/migrate/20170215205445_create_balances.rb

```diff
@@ -2,6 +2,8 @@ class CreateBalances < ActiveRecord::Migration
   def change
     create_table :balances do |t|
       t.float :amount, default: 0, null: false
+      t.float :money_charged, default: 0, null: false
```

Comment:
Don't things like that. You changed migration, but this migration was already applied! So these changes will not be applied. Instead of this approche just create new migration, that will add money_charged, money_spent to the `balance`